src/Castle.ts

```diff
@@ -99,7 +105,37 @@ Request: ${JSON.stringify(requestOptions)}
 ${response ? responseFormatter(response, body) : errorFormatter(err)}
 `;
 
+// Although headers are technically typed `string | string[]`
+// the only header that exists as an `array` is 'set-cookie'.
+// It is safe to cast all other headers to strings.
+const getCfConnectingIp = (request: ExpressRequest) =>
+  request.headers['cf-connecting-ip'] as string;
+const getForwardedFor = (request: ExpressRequest): string => {
+  const rf = request.headers['x-forwarded-for'] as string;
+  return rf && rf.split(',')[0].trim();
+};
+const getCidHeader = (request: ExpressRequest) =>
+  request.headers['x-castle-client-id'] as string;
```

Comment:
is 'as string' more recommended than <string> ?

Reply:
`as` is recommended but both work and have the same effect. Seems like `<>` was the old way, they added `as` to remove ambiguity